packages/circle-demo-webapp/app/tailwind.css

```diff
@@ -8,95 +8,18 @@
     --background: 0 0% 100%; /* White background for light mode */
     --foreground: 240 10% 3.9%; /* Dark text for light mode */
 
-    /* Card and Popover Colors */
-    --card: 0 0% 100%;
-    --card-foreground: 240 10% 3.9%;
-    --popover: 0 0% 100%;
-    --popover-foreground: 240 10% 3.9%;
-
     /* Primary Brand Color - Jelly 400 */
     --primary: 255 82% 64%; /* Jelly 400 - Vibrant purple */
```

Comment:
I left these as an example. Changing the primary color here (in the demo app) will override the primary color from circle-elements, which is the expected and intended behavior.